CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v0.9.0 (Unreleased)
+#### BUG FIXES
+* `pingfederate_oauth_client` resource has the following bugs resolved ([#221](https://github.com/pingidentity/terraform-provider-pingfederate/pull/221)):
+  - `client_id` now correctly forces a replacement of the resource. Previously if the resource was created, then the `client_id` value was changed, would result in `"Error deleting client. Client <client_id> not found"}`
+  - `Error: client_auth.secret cannot be empty when "CLIENT_CREDENTIALS" is included in grant_types.` when supplying sensitive attribute
```

Comment:
It would be better to describe the bug fixes here than copying the error text. Like "fixed unexpected new value error when setting restrict_scopes to false" or whatever.